src/components/GlobalPreferences/Network/Network.js

```diff
@@ -29,6 +30,7 @@ function Network({ wrapper }) {
     network,
   } = useNetwork(wrapper)
   const theme = useTheme()
+  const { layoutName } = useLayout()
```

Comment:
I thought `useLayout` would not work in the client? cc @bpierre 
Should this use `useViewport` instead?

Reply:
But we do declare `Layout` in the Preferences, as various components (e.g. Box, Split) need them.